projects/components/src/filtering/filters/numeric/numeric-filter-content-renderer.component.ts

```diff
@@ -1,6 +1,6 @@
 import { ChangeDetectionStrategy, Component, Inject } from '@angular/core';
 import { FilterOperatorType } from '../../model/filter-operator-type';
-import { FILTER_RENDERER_API, FilterRendererApi } from '../../rendering/filter-renderer.component';
+import { FilterRendererApi, FILTER_RENDERER_API } from '../../rendering/filter-renderer.component';
```

Comment:
Hmm do we not lint test files? How'd this get by CI

Reply:
this whole directory is defunct, it was replaced by filter bar. Will remove it in a follow up.